### ISSUE TITLE
Azure Blob Naming requirements too vague

### ISSUE THREAD

**Issue by dcolliervb23** [MicrosoftDocs/feedback#784]
**Describe the bug**
The blob naming reference states "some ASCII or Unicode characters.... are also not allowed" then points us to a huge RFC doc.  This guidance is overly broad and the referenced RFC doc is large and hard to parse.  The statement says "some" so we are not even sure if we have to exactly match what is in the RFC doc or just playing a long guessing game.  Any additional detail on exactly what we need to validate for would be very helpful.  FYI, the [NameValidator.ValidateBlobName](https://docs.microsoft.com/en-us/dotnet/api/microsoft.windowsazure.storage.namevalidator.validateblobname?view=azure-dotnet) does not validate for control characters or any of these other "some" characters.

**To Reproduce**
1. Go to blob naming standard page [here](https://docs.microsoft.com/en-us/rest/api/storageservices/naming-and-referencing-containers--blobs--and-metadata) then scroll to the grey important box.  
2. Read the second to last sentence 

> some ASCII or Unicode characters, like control characters (0x00 to 0x1F, \u0081, etc.), are also not allowed

**Expected behavior**
Specific guidance on exactly what we need to be validating for.

**Comment by Powerhelmsman** [comment]
@tamram Can you take a look at this suggestion?

**Comment by Powerhelmsman** [comment]
@CamSoper Can you take a look at this issue?
